docs/installation.rst

```diff
@@ -50,15 +50,16 @@ You can install starknet.py on Windows in two ways:
 
 1. Install it just like you would on Linux.
 
-You might encounter problems related to ``libcrypto_c_exports``.
-
 In such case make sure that you have `MinGW <https://www.mingw-w64.org/>`_ installed and up-to-date.
 
 .. hint::
-    An easy way to install MinGW is through `chocolatey <https://community.chocolatey.org/packages/mingw>`_.
+    The recommended way to install is through `chocolatey <https://community.chocolatey.org/packages/mingw>`_.
 
     You also should have MinGW in your PATH environment variable (e.g. ``C:\ProgramData\chocolatey\lib\mingw\tools\install\mingw64\bin``).
 
+.. warning::
+    You might encounter problems related to ``libcrypto_c_exports`` (e.g  LoadLibraryEx).
+    Install via chocolatey and set proper PATH for it should solve your problem.
```

Comment:
```suggestion
.. warning::
    Please be aware that you may encounter issues related to ``libcrypto_c_exports`` (e.g  LoadLibraryEx).
    Installing MinGW via chocolatey and correctly adding  it to the PATH should solve these issues.
```
I would rephrase it slightly to explicitly mention that it's still about the installation of `MinGW`. Wdyt?